create_framework/introduction.rst

```diff
@@ -62,7 +62,7 @@ Before You Start
 Reading about how to create a framework is not enough. You will have to follow
 along and actually type all the examples included in this tutorial. For that,
 you need a recent version of PHP (5.5.9 or later is good enough), a web server
-(like Apache, NGINX or PHP's built-in web server), a good knowledge of PHP and
+(like Apache, nginx or PHP's built-in web server), a good knowledge of PHP and
```

Comment:
**Nginx** (stylized as **NGINX** or **nginx** or **NginX**).

Reply:
@Arman-Hosseini as you said, "nginx" can be spelled in multiple forms ... so we decided to use `nginx` everywhere in the Symfony Docs and that's why we're changing this.